Packages/Tracking/OpenXR/API Layer/HandTrackingApiLayerFeature.cs

```diff
@@ -0,0 +1,116 @@
+using JetBrains.Annotations;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using UnityEngine.XR.OpenXR;
+using UnityEngine.XR.OpenXR.Features;
+
+#if UNITY_EDITOR
+using System;
+using UnityEditor;
+using UnityEditor.XR.OpenXR.Features;
+using UnityEditor.Build.Reporting;
+#endif
+
+namespace Ultraleap.Tracking.OpenXR.ApiLayer
+{
+    /// <summary>
+    /// Embeddable Ultraleap OpenXR API Layer.
+    /// </summary>
+#if UNITY_EDITOR
+    [OpenXRFeature(FeatureId = FeatureId,
+        Version = "1.0.0",
+        UiName = "Ultraleap Hand Tracking API Layer",
+        Company = "Ultraleap",
+        Desc = "Embeddable API layer for Ultraleap Hand Tracking",
+        Category = FeatureCategory.Feature,
+        Required = false,
+        OpenxrExtensionStrings = "",
+        BuildTargetGroups = new[] { BuildTargetGroup.Android }
```

Comment:
I wonder if we can name this feature (and possibly the existing one) more descriptively?

currently it's hard to know what each do via the UI alone
![image](https://user-images.githubusercontent.com/60962475/203833721-2f4b9c85-d4b1-4591-8e48-dcaa4b4946b3.png)

Reply:
I'm happy to entertain suggestions. Most other vendors just have "Hand Tracking" as the feature name. I added the vendor to make it clearer but that might not be needed now it's part of a feature set.